Rubberduck.Parsing/ComReflection/ReferencedDeclarationsCollector.cs

```diff
@@ -244,6 +248,47 @@ private static Attributes GetModuleAttributes(IComType module)
             }
         }
 
+        private void CreatePropertyDeclarations(IEnumerable<ComField> properties, QualifiedModuleName moduleName, Declaration declaration,
+            SerializableDeclarationTree moduleTree)
+        {
+            foreach (var item in properties.Where(x => !x.Flags.HasFlag(VARFLAGS.VARFLAG_FRESTRICTED)))
+            {
+                Debug.Assert(item.Type == DeclarationType.Property);
+                var attributes = GetPropertyAttibutes(item);
+
+                var getter = new PropertyGetDeclaration(item, declaration, moduleName, attributes);
+                _declarations.Add(getter);
+                var propertyTree = new SerializableDeclarationTree(getter);
+                moduleTree.AddChildTree(propertyTree);
+
+                var coClass = getter.ParentDeclaration as ClassModuleDeclaration;
+                if (coClass != null && attributes.HasDefaultMemberAttribute())
+                {
+                    coClass.DefaultMember = getter;
+                }
+
+                if (item.Flags.HasFlag(VARFLAGS.VARFLAG_FREADONLY))
+                {
+                    continue;
+                }
+
+                if (item.IsReferenceType)
```

Comment:
This is impossible. The short form declaration is basically the same as a public field, so in it's native form there isn't a set of accessors - only a read-only flag. The way they are used by the consumer is language specific. This may require more testing as to how the VB compiler handles them, but I can't make that determination until I find one that is actually a reference type.